Documentation/security/policy/language.rst

```diff
@@ -39,10 +39,10 @@ can talk to each other. Layer 3 policies can be specified using the following me
   above. DNS information is acquired by routing DNS traffic via a proxy.
   DNS TTLs are respected.
 
-.. _Labels based:
+.. _Endpoints based:
 
-Labels Based
-------------
+Endpoints Based
+---------------
 
 Label-based L3 policy is used to establish policy between endpoints inside the
 cluster managed by Cilium. Label-based L3 policies are defined by using an
```

Comment:
There are two more references to `label-based` here, do these need to change as well?

Reply:
Ah good point, yes let me fix that up.